public/components/event_analytics/explorer/explorer.tsx

```diff
@@ -1503,7 +1503,6 @@ export const Explorer = ({
           stopLive={stopLive}
           setIsLiveTailPopoverOpen={setIsLiveTailPopoverOpen}
           liveTailName={liveTailNameRef.current}
-          searchError={explorerVisualizations}
```

Comment:
what's up with removing searcherror?

Reply:
we don't need the search error any more from here. More info list in below comment.